src/ir_Trotec.h

```diff
@@ -17,7 +20,8 @@
 #define TROTEC_DRY            2
 #define TROTEC_FAN            3
 
-#define TROTEC_ON        1 << 3
```

Comment:
Honestly, I think your original approach was nicer.
No need to change it. Just saying.
e.g.
```
#define TROTEC_ON        (1 << 3)
```